RecoTracker/PixelSeeding/plugins/alpaka/BrokenLineFit.dev.cc

```diff
@@ -159,11 +159,13 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         }
         brokenline::fastFit(acc, hits, fast_fit);
 
+#ifdef BROKENLINE_DEBUG
         // no NaN here....
         ALPAKA_ASSERT_ACC(fast_fit(0) == fast_fit(0));
         ALPAKA_ASSERT_ACC(fast_fit(1) == fast_fit(1));
         ALPAKA_ASSERT_ACC(fast_fit(2) == fast_fit(2));
         ALPAKA_ASSERT_ACC(fast_fit(3) == fast_fit(3));
```

Comment:
Could you change these to
```suggestion
        ALPAKA_ASSERT_ACC(not alpaka::math::isnan(acc, fast_fit(0)));
        ALPAKA_ASSERT_ACC(not alpaka::math::isnan(acc, fast_fit(1)));
        ALPAKA_ASSERT_ACC(not alpaka::math::isnan(acc, fast_fit(2)));
        ALPAKA_ASSERT_ACC(not alpaka::math::isnan(acc, fast_fit(3)));
```
to make the meaning of the checks more explicit ?

Reply:
implemented in the last push.